src/_includes/partials/contact/contact-us-box.html

```diff
@@ -0,0 +1,9 @@
+<div class="[ box ] [ box__equal-space ]">
```

Comment:
`box__equal-space` to `box--equal-space`

For the BEM style (Block-Element-Modifier just to be clear) of class names, we use `__` for child elements and `--` for modifications to the same element. We're doing the second one here.